apps/ledger-live-mobile/src/families/stellar/AddAssetFlow/03-ValidationSuccess.tsx

```diff
@@ -44,7 +46,11 @@ export default function ValidationSuccess({ navigation, route }: Props) {
       account && account.type === "Account"
         ? listTokensForCryptoCurrency(account.currency)
         : [];
-    return options.find(({ id }) => id === transaction.assetId);
+    return options.find(
+      ({ tokenType, contractAddress }) =>
+        tokenType === transaction.assetCode &&
```

Comment:
Are we sure this is equivalent ? Was it even working before ?

Reply:
@elbywan @hedi-edelbloute from what I can remember, I asked @lambertkevin and I think that what I was told to replase the assetId. 

@lambertkevin can you approve what I just said ?

Reply:
My bad, @henrily-ledger was the one who helped me.